runtime/src/dpe_platform.rs

```diff
@@ -96,26 +101,29 @@ impl Platform for DpePlatform<'_> {
         Ok(issuer_len)
     }
 
-    fn get_issuer_sn(&mut self, out: &mut [u8; MAX_SN_SIZE]) -> Result<usize, PlatformError> {
-        let sn = self.hashed_rt_pub_key.bytes();
-        let size = min(MAX_SN_SIZE, sn.len());
-        // Prevents potential panic
-        if size > out.len() || size > sn.len() {
-            return Err(PlatformError::IssuerNameError(0));
+    fn get_signer_identifier(&mut self) -> Result<SignerIdentifier, PlatformError> {
+        let mut ski = [0u8; MAX_SKI_SIZE];
+        let hashed_rt_pub_key = self.hashed_rt_pub_key.bytes();
```

Comment:
nit: Can you link to the code in `x509/` that generates the RT SKI?

Reply:
Done